src/main/webapp/WEB-INF/views/car/create.jsp

```diff
@@ -0,0 +1,30 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create car</title>
+    <style>
+        body {
+            text-size-adjust: 100%;
+            box-sizing: border-box;
+            margin: 5px;
+            font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+            font-size: 14px;
+            line-height: 1.5;
+            color: rgb(36, 41, 46);
+            background-color: rgb(255, 255, 255);
+        }
+        button {
+            background-color: white;
+        }
+    </style>
+</head>
+<body>
+<h3>Enter values for car:</h3>
+<form method="post" action="${pageContext.request.contextPath}/cars/add">
+    Model <input type="text" name="model"><br>
+    Manufacturer id <input type="text" name="manufacturerId"><br>
+    Driver id <input type="text" name="driverId"><br>
```

Comment:
check that id is number in all places
also:
<img width="577" alt="image" src="https://user-images.githubusercontent.com/58032907/203313329-f2f77b2d-6a53-4559-8db9-e4546127b235.png">